src/Workspaces/Core/Portable/CodeStyle/NotificationOption.cs

```diff
@@ -14,17 +17,29 @@ public class NotificationOption
     {
         public string Name { get; set; }
 
-        public DiagnosticSeverity Value { get; set; }
+        public ReportDiagnostic Severity
+        {
+            get;
```

Comment:
Why did the accessor formatting change?

Reply:
➡️ It's not specified in .editorconfig, and this is a new property so there wasn't a format to preserve. The formatting for the existing property `Value` changed because it needed expression-bodied accessors.